src/main/java/com/code_intelligence/jazzer/mutation/mutator/jazzer/FuzzedDataProviderMutatorFactory.java

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Having thought more about it, I would propose to delete this file. If we make `FuzzedDataProvider` a regular supported type, users can create multiple instances per fuzz test with potentially confusing semantics.

Instead, I think that we should keep support of this for backwards compatibility but shouldn't expose it in the new mutation framework.

Reply:
Agreed, sounds reasonable.